Stratis.Bitcoin/Builder/FullNodeBuilder.cs

```diff
@@ -49,8 +49,9 @@ public FullNodeBuilder(NodeSettings nodeSettings)
 			List<Action<IFeatureCollection>> featuresRegistrationDelegates, IFeatureCollection features)
 			: this(configureServicesDelegates, configureDelegates, featuresRegistrationDelegates, features)
 		{
-			this.NodeSettings = nodeSettings ?? NodeSettings.Default();
-			this.Network = nodeSettings.GetNetwork();
```

Comment:
Even for the public setter its fine to set in the constructor the public setter is caller after the constructor is executes.
So just fix this and lets commit